### ISSUE TITLE
Collections Not Available For Selection in Workform Relationships Tab

### ISSUE THREAD

**Issue by KevinJonesMeta** [OregonDigital/OD2#2877]
### Descriptive summary

When ingesting a new work in the form, in the Relationships tab the Collections search box does not return known collections when using the collection label, the collection ID, or the public url for the collection.

### Expected behavior

When using type-ahead search in Collections box in Relationships tab to add a work to collections, the collection is available when searching by collection label.

### Related work

I did not find a related ticket.

**Comment by straleyb** [comment]
@KevinJonesMeta This COULD be a code issue depending on what the needs are for this. I discovered that this looks to be a permissions issue. I added myself as a depositor to the collection (Oregon Multicultural Archive) and was able to find the collection when I used the autocomplete on a new work I was creating. The search that gets fired off has the depositor group attached to it, so it will only return results that match someone being a depositor for that collection.

**Comment by KevinJonesMeta** [comment]
@straleyb Is there a permission level that includes depositing on all collections or all collections for a given institution? I would prefer it was a simple selection list but there could be use cases I'm not aware of for limiting permission to a group of depositors. 

Also, when I create or edit a work the collections I can access in autocomplete include a variety of UO collections as well as a few OSU collections. The difference I have seen is that the collections I can add to through the form were created after migration but I can't say that for sure.

**Comment by straleyb** [comment]
Im not sure about use cases either. If this functionality needs to be changed, I can do that, so once we know for sure. One thing I did notice is that some collections I checked have Repository Managers and admin under the managers. The one I looked at did not. So if they did get migrated, they may not have been attributed those. That could be what is causing them to not show up even if you are admin/depositor
<img width="1566" alt="Screen Shot 2023-09-19 at 12 38 33 PM" src="https://github.com/OregonDigital/OD2/assets/6424683/01866bc3-20a9-4901-8328-3d3c2cc6b344">

**Comment by KevinJonesMeta** [comment]
@straleyb Is the manager list above hierarchical? That would be curious if it is.

We talked about it in Metadeities and the use case for limiting collection selection is so that not all users can place works anywhere, so some quality control. We looked at user roles and workflow roles and found that I was an administrator but did not have any other workflow roles assigned so we thought the Admin role was not triggering full access to collections in the collection lookup in the form (this may be the same as what you said above). So as a starting point, can we make the Admin role automatically be a manager on all sets? Or, alternatively, make the admin role a universal manager in the collection lookup? I don't know if those are separate functions.
